src/main/java/org/elasticsearch/cluster/ClusterState.java

```diff
@@ -600,6 +647,13 @@ public ClusterState build() {
             return os.bytes().toBytes();
         }
 
+        public static byte[] toDiffBytes(ClusterState previousState, ClusterState state) throws IOException {
```

Comment:
ok I think this is only used in LocalDiscover and in a test, can we move the method to LocalDiscovery instead?